asv_bench/benchmarks/index_object.py

```diff
@@ -194,5 +194,8 @@ def setup(self, N):
     def time_monotonic_inc(self, N):
         self.intv.is_monotonic_increasing
 
+    def time_is_unique(self, N):
```

Comment:
should also have an index that has a duplicate in it

Reply:
The duplicated element is (0,1).